src/ucs/datastruct/usage_tracker.h

```diff
@@ -162,4 +164,14 @@ ucs_usage_tracker_touch_key(ucs_usage_tracker_h usage_tracker, void *key)
     ucs_lru_push(usage_tracker->lru, key);
 }
 
+/**
+ * @brief Checks if a key is promoted.
+ *
+ * @param [in]  usage_tracker  Handle to the Usage Tracker object.
+ * @param [in]  key            Key of the entry to be checked.
+  
+ * @return 1 if entry is promoted, 0 otherwise.
+ */
+int ucs_usage_tracker_is_promoted(ucs_usage_tracker_h usage_tracker, void *key);

```

Comment:
where is it used? where is it tested?

Reply:
used in when we select a transport and we want only "important" transports like RC